### ISSUE TITLE
Cmake version

### ISSUE THREAD

**Issue by danielkucera** [JF002/Pinetime#15]
I'm getting following error:
```
$ cmake ../
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  CMake 3.15 or higher is required.  You are running version 3.10.2
```
Is such a new version of cmake necessary? I'm on fully updated `Ubuntu 18.04 LTS`

**Comment by JF002** [comment]
That's the version that is used by my IDE. You should be able to edit those CMakeList.txt files to lower the requirements.

Tell me if it works with 3.10, then I'll commit the changes :)

**Comment by JF002** [comment]
Fixed in #17
